setup.py

```diff
@@ -60,6 +60,24 @@
         "blendsearch": ["flaml[blendsearch]"],
         "mathchat": ["sympy", "pydantic==1.10.9", "wolframalpha"],
         "retrievechat": ["chromadb", "sentence_transformers", "pypdf", "ipython", "beautifulsoup4", "markdownify"],
+        "retrievechat-pgvector": [
+            "pgvector>=0.2.5",
+            "psycopg>=3.1.18",
+            "sentence_transformers",
+            "pypdf",
+            "ipython",
+            "beautifulsoup4",
+            "markdownify",
+        ],
+        "retrievechat-qdrant": [
+            "flaml[automl]",
```

Comment:
flaml[automl] shouldn't be part of this option.

Reply:
This was cleaned up.